pages/common/fswatch.md

```diff
@@ -0,0 +1,7 @@
+# fswatch
+
+> A cross-platform file change monitor.
+
+- Run a single bash command on file addition, update or deletion. Use `{}` in your bash command as a placeholder for the absolute path to the file change.
```

Comment:
Could the second sentence be used in the main description, above?

Also, instead of "file addition", I think "file creation" is more intuitive.